### DISCUSSION TITLE
"Rest" ignored in experiment

### DISCUSSION THREAD

**Discussion by sklein-ise** [pybamm-team/PyBaMM#1531] [Q&A] [answered]
Hello,

I noticed that when using a parameter set with the experiment class the "Rest" command does not work. It keeps discharging the battery by the set "current function" value. Is this a mistake or am I doing something wrong? 

Thanks in advance!
Siri

import pybamm
import matplotlib.pyplot as plt

experiment = pybamm.Experiment(
    [("Rest for 50 minutes")] * 1,
)

model = pybamm.lithium_ion.DFN()

param = model.default_parameter_values
param.process_model(model)

sim = pybamm.Simulation(model, experiment=experiment)
sim.solve()

solution = sim.solution 
   
fig = plt.figure("Time")
t_sol = solution["Time [s]"].entries
plt.plot(t_sol, solution["Terminal voltage [V]"](t_sol),'--', color = 'red', label = 'gitt NMC_gra')
plt.xlabel("Time [s]")
plt.ylabel("Voltage [V]")
plt.tight_layout()
plt.show()

**Comment by brosaplanella** [comment, marked as answer]
Hi Siri! I think the issue is that you processed the model with the parameters. Try instead running this:
```python3
import pybamm
import matplotlib.pyplot as plt

experiment = pybamm.Experiment(
[("Rest for 50 minutes")] * 1,
)

model = pybamm.lithium_ion.DFN()

param = model.default_parameter_values

sim = pybamm.Simulation(model, parameter_values=param, experiment=experiment)
sim.solve()

solution = sim.solution
```

Note that because you are using the default parameter values you could just not define them and run the simulation as
```python3
sim = pybamm.Simulation(model, experiment=experiment)
```
(if not specified it uses the default parameters).

**Comment by sklein-ise** [comment]
Thanks! Now it works. I didn't realise you don't need to process the model after updating the parameters.
